src/components/AboutUsAddMember/AboutUsAddMember.tsx

```diff
@@ -0,0 +1,118 @@
+import React, { useRef } from 'react'
+import { FormProvider, useForm } from 'react-hook-form'
+import { useTranslation } from 'react-i18next'
+import { ICreateTeamMember } from '@allTypes/reduxTypes/aboutUsStateTypes'
+import { Translation } from '@constants/translations'
+import { yupResolver } from '@hookform/resolvers/yup'
+import { dispatch } from '@redux/hooks'
+import { aboutUsMiddleware } from '@redux/slices/aboutUs'
+import { Button } from '@uiComponents/Button'
+import FileField from '@uiComponents/FileField/FileField'
+import TextField from '@uiComponents/FormFields/TextField'
+import { createAboutUsValidationSchema } from '@validation/aboutUs/aboutUs'
+import Image from 'next/image'
+
+import { useImageUpload } from '../../hooks/ImageUpload'
+
+const defaultValues = {
+  header: '',
+  paragraph: '',
+  imageDescription: '',
+  imageURL: null,
+}
+const AboutUsAddMember = () => {
+  const { imageLoaded, handleFileChange } = useImageUpload()
```

Comment:
File should be AboutUs/AddMember/AboutUsMember.tsx

Reply:
done